inspire/base/templates/format/record/Inspire_HTML_detailed_macros.tpl

```diff
@@ -328,4 +328,24 @@
       <a class="btn btn-default pull-right" href="/search?p=refersto%3Arecid%3A+{{record['control_number']}}&cc=HEP" data-toggle="modal">View in Search Results</a>
     </div>
   </div>
+{% endmacro %}
+
+{% macro mathjax() %}
+<script src="//cdnjs.cloudflare.com/ajax/libs/mathjax/2.5.3/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>
```

Comment:
It's better to move out the 2.5.3 version, so that we can easily upgrade MathJax.